src/components/GuideTile/GuideTile.js

```diff
@@ -13,6 +20,8 @@ const GuideTile = ({ duration, title, description, className, to }) => (
     css={css`
       display: grid;
       grid-template-rows: auto 1fr auto;
+      width: ${procIdxIsInteger ? '100%' : '90%'};
+      margin-left: ${procIdxIsInteger ? '0' : '10%'};
```

Comment:
changing the styles here have had some unintended consequences in other places that we use the GuideTiles. I think moving these styles to the LabOverviewTemplate might be best. you can use the css prop within that guidetile just like here and select for the p and h3

normal styling:
<img width="998" alt="Screen Shot 2022-02-04 at 9 15 26 AM" src="https://user-images.githubusercontent.com/39655074/152574381-9fd1636c-2ec9-4ee3-a000-fe4e9b567b44.png">

current changes:
<img width="1009" alt="Screen Shot 2022-02-04 at 9 15 34 AM" src="https://user-images.githubusercontent.com/39655074/152574409-2c4ca1e0-ca7f-46c3-93e1-b05e5699f16e.png">

Reply:
That spacing between the tiles seems really wide I'd recommend following the pattern we use for space between the quickstart tiles on I/O 

https://developer.newrelic.com/instant-observability/
<img width="1420" alt="Screen Shot 2022-02-04 at 11 00 32 AM" src="https://user-images.githubusercontent.com/1395158/152579642-7f6814b3-1039-43c7-b0db-4ece97d27e7a.png">

Reply:
Totalny agree, I didn’t noticed that change will have so big impact on other views, i will move that styles to LabOverviewTemplate asap to keep website pattern, thank you for your advices

Reply:
Thanks @kisielewskik!